Public/Src/Sandbox/Windows/DetoursServices/DetoursServices.h

```diff
@@ -129,3 +129,29 @@ class TranslatePathTuple
         return fromPath;
     }
 };
+
+// CODESYNC: AnyBuildShimInfo.cs :: ShimProcessMatch class
+class ShimProcessMatch
+{
+public:
+    std::unique_ptr<wchar_t> ProcessName;

```

Comment:
>std::unique_ptr<wchar_t> ProcessName; [](start = 4, length = 37)

Any reason why not just use std::wstring ?

Reply:
Moved to wstring